### DISCUSSION TITLE
should I mix install?

### DISCUSSION THREAD

**Discussion by gituser23234235234887** [FiloSottile/age#559] [Q&A]
Hello. I want the newest `age` binaries, but also want `apt` update and the man pages.

my idea on how to achieve this is to install `age` through `apt`, and then replace the binaries with the ones on `github`'s latest release. this way I get to have man pages, and if `apt` deems a security update to be necessary, then I will get that update; while in the meantime I use version `1.1.1` instead of the current `1.0.0` that comes with `apt jammy`.

I am wondering if anyone recommends I avoid doing this, and if there is a better way for me to achieve what I want.

**Comment by mohammed90** [comment]
Use `dpkg-divert` command. It'll be something like this:

```
sudo dpkg-divert --divert /usr/bin/age.default --rename $(which age)
sudo mv ./<latest age binary> /usr/bin/age.custom
sudo update-alternatives --install /usr/bin/age age /usr/bin/age.default 10
sudo update-alternatives --install /usr/bin/age age /usr/bin/age.custom 50
```
